charts/operator/values.yaml

```diff
@@ -307,7 +295,7 @@ initChownData: true
 ##
 persistence:
   enabled: true
-  storageClassName: ""
+  storageClassName: "standard"
```

Comment:
Maybe it's better to not specify any value here so that a default storage class will be used?